build.gradle

```diff
@@ -80,7 +80,7 @@ task javadocJar(type: Jar) {
     from javadoc.destinationDir
 }
 
-version '1.0.0.0'
+version '1.0.0'
```

Comment:
Super minor nit sorry about that.
We would need the 4th digit for minor versions.